### ISSUE TITLE
Update keycloak clients

### ISSUE THREAD

**Issue by pameladelgado** [SwissDataScienceCenter/renku-admin-docs#13]
Update keycloak clients to reflect 

- [ ] https://github.com/SwissDataScienceCenter/renku/pull/662 (check if needed) and

- [ ] https://github.com/SwissDataScienceCenter/renku/pull/677

**Comment by rokroskar** [comment]
Presumably this has been done?

**Comment by pameladelgado** [comment]
not really, but we need to update other things in the repo as well, like the values file.

**Comment by rokroskar** [comment]
You mean that these changes have been done in our charts but not in our long-running deployments? Maybe clarify which keycloak instances need to be updated?

**Comment by pameladelgado** [comment]
The changes have been done both in our charts and in our deployments. We just need to update [these](https://github.com/SwissDataScienceCenter/renku-admin-docs/tree/master/keycloak-standalone-clients) in case somebody wants to use an existing keycloak instance to deploy Renku.
